### PR TITLE
Monthly Contributions Thank You page

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

The thank you page for monthly contributions. It's not amazingly beautiful right now, but it's better than the 404 that #87 is linking to.

This is mainly a placeholder for now, so that we have something working for the whole flow; it will be tweaked when we apply some UXD to it.

[**Trello Card**](https://trello.com/c/bthrjAkK/607-create-a-thank-you-page-for-monthly-contributions)

## Changes

- Added thank you page files.
- Added route for thank you page.
- Added page to webpack.

## Screenshots

**Mobile:**

thanks-mobile

**Desktop:**

thanks-wide

### CODE REVIEW

File: assets/pages/monthly-contributions-thankyou/monthlyContributionsThankyou.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import SimpleHeader from 'components/headers/simpleHeader/simpleHeader';
+import SimpleFooter from 'components/footers/simpleFooter/simpleFooter';
+
+import * as ga from 'helpers/ga';
+import * as abTest from 'helpers/abtest';
+import * as logger from 'helpers/logger';
+
+
+// ----- AB Tests ----- //
```
Comment (svillafe) [inline assets/pages/monthly-contributions-thankyou/monthlyContributionsThankyou.jsx]: We should extract this common chunk asap in a helper module. We now have this piece of code 3 times across the site.

**Comment by Ap0c** [comment]
Waiting to merge this until we clarify the issue with #87.

File: assets/pages/monthly-contributions-thankyou/monthlyContributionsThankyou.jsx (hunk 1 above)
Comment (Ap0c) [inline assets/pages/monthly-contributions-thankyou/monthlyContributionsThankyou.jsx]: I've carded this up.

**Comment by Ap0c** [comment]
It's appeared on PROD, teamcity was taking a while, but it looks fine.

**Pull request merged**